packages/di/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './decorators';
+export * from './models';
+export * from './services';
```

Comment:
It would be good to review what we export and make sure the library does not expose unnecessary things that the consumer would not use.

![image](https://user-images.githubusercontent.com/18616324/149886091-dec094c5-1824-414a-949e-e0ab35dee6a2.png)